### PR TITLE
Remove dependencies on DCI flags

### PR DESCRIPTION
**Pull request by betoredhat**
Using cluster information for storage configuration

### CODE REVIEW

**Review by manurodriguez — APPROVED** [review]
Thanks, LGTM

File: roles/odf_setup/tasks/local-storage-operator.yml [hunk 1]
```diff
@@ -73,5 +73,5 @@
             annotations:
               "{{ default_storageclass_annotation }}"
   when:
-    - not enable_ocs | default(false) | bool
+    - ocs_install_type == 'internal'
```
Comment (ramperher) [inline roles/odf_setup/tasks/local-storage-operator.yml:76]: you're already checking this in main.yml, do you need it here?

Comment (betoredhat) [reply roles/odf_setup/tasks/local-storage-operator.yml:76]: yes, not needed. Thanks!

File: roles/label_nodes/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  - An Openshift cluster.
+  - Access to the kubeconfig file via the `KUBECONFIG` environment variable
+  - The inventory host should have the labels needed in JSON format
+    ```json
+    [workers:vars]
+    labels={"Location": "Boston", "owner": "OCP operators", "cost_center", "IT"} 
+
+    [ocs_nodes:vars]
+    labels={"cluster.ocs.openshift.io/openshift-storage": ""}    
+    ```
```
Comment (tonyskapunk) [inline roles/label_nodes/README.md:25]:
```suggestion
```toml
[workers:vars]
labels={"Location": "Boston", "owner": "OCP operators", "cost_center", "IT"} 

[ocs_nodes:vars]
labels={"cluster.ocs.openshift.io/openshift-storage": ""}    
```
```

**Review by manurodriguez — COMMENTED** [review]
Thanks Beto, now that you did some clean up in the labels and it's more clear,  can we also include the `label_nodes` role in the Dependencies of the README[1] 

[1] https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/main/roles/odf_setup/README.md?plain=1#L89

**Comment by betoredhat** [comment]
> Thanks Beto, now that you did some clean up in the labels and it's more clear, can we also include the `label_nodes` role in the Dependencies of the README[1]
> 
> [1] https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/main/roles/odf_setup/README.md?plain=1#L89

Done

**Pull request merged**